apps/condo/migrations/20230214175344-0221_billingreceipthistoryrecord_invalidserviceserror.js

```diff
@@ -0,0 +1,26 @@
+// auto generated by kmigrator
+// KMIGRATOR:0221_billingreceipthistoryrecord_invalidserviceserror:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMS4zIG9uIDIwMjMtMDItMTQgMTI6NTMKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAyMjBfYXV0b18yMDIzMDIxM18xMDU1JyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiaWxsaW5ncmVjZWlwdGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdpbnZhbGlkU2VydmljZXNFcnJvcicsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5KU09ORmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
```

Comment:
BillingReceipt is the (perhaps?) biggest table in all condo. So this migration would fail on production. 

The reason for failing would be migration time exceeding the time limit set by the postgres on SQL operations (i think its 10 seconds or so....) 

You need to add a bigger time limit in the beginiing of migration and set it back to 10 seconds in the end of migration. 

Same applies to exports.down migration as well...

Please check out this migration for example: apps/condo/migrations/20220727181205-0156_ticket_lastcommentat_ticketchange_lastcommentatfrom_and_more.js